### PR TITLE
Player initial

### PR DESCRIPTION
**Pull request by DmIvanov**
- fixes in project settings to run the project on a real device
- the first working implementation of Player
- Organisation name changed to "Grid Cascade"

### CODE REVIEW

File: src/Player/RendererViewContextEAGL.swift [hunk 1]
```diff
@@ -0,0 +1,18 @@
+//
+//  RendererViewContextEAGL.swift
+//  Smena
+//
+//  Created by Dmitrii on 07/04/2018.
+//  Copyright © 2018 Andrei Valkovskii. All rights reserved.
```
Comment (valkovsky) [inline src/Player/RendererViewContextEAGL.swift]: We should change copyright here as well.

Comment (DmIvanov) [reply src/Player/RendererViewContextEAGL.swift]: done. changed for all files which are already in the project

File: src/Player/LowLevelRendererViewContext.swift [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+import CoreImage
+
+
+class LowLevelRendererViewContext {
+
+    let ciContext: CIContext
+
+    init(ciContext context: CIContext?) {
+        if context == nil {
```
Comment (valkovsky) [inline src/Player/LowLevelRendererViewContext.swift:17]: Could be a one-liner:
```swift
 self.ciContext = context ?? CIContext(options:nil)
```

**Review by valkovsky — APPROVED** [review]
Awesome stuff!

File: src/Player/LowLevelRendererViewContext.swift (hunk 2 above)
Comment (DmIvanov) [reply src/Player/LowLevelRendererViewContext.swift:17]: Have you heard about the impact of the ternary operator on compile time in Swift? ;-)

Comment (valkovsky) [reply src/Player/LowLevelRendererViewContext.swift:17]: No, I haven't. Do you have problems with compiling this project? :)

**Pull request merged**